Source/Core/Core/FifoPlayer/FifoPlayer.cpp

```diff
@@ -163,12 +163,16 @@ void FifoPlaybackAnalyzer::OnCommand(const u8* data, u32 size)
 
 bool IsPlayingBackFifologWithBrokenEFBCopies = false;
 
-FifoPlayer::FifoPlayer() : m_Loop{Config::Get(Config::MAIN_FIFOPLAYER_LOOP_REPLAY)}
+FifoPlayer::FifoPlayer()
+    : m_Loop{Config::Get(Config::MAIN_FIFOPLAYER_LOOP_REPLAY)},
+      m_EarlyMemoryUpdates{Config::Get(Config::MAIN_FIFOPLAYER_EARLY_MEMORY_UPDATES)}
 {
+  m_config_changed_callback_id = Config::AddConfigChangedCallback([this] { RefreshConfig(); });
 }
```

Comment:
Add a call to `RefreshConfig` after registering the callback to ensure that the correct setting values are available even if the user never changes the config during the lifetime of the FifoPlayer.

Reply:
That's not needed since we initialize the values from the config right above - I could get rid of that initialization, but then I'd want to set default values in FifoPlayer.h to avoid uninitialized variable warnings.

Reply:
Changed, with default values in the header and no explicit setting in the constructor other than the call to `RefreshConfig`.